src/Compiler/Service/ServiceCompilerDiagnostics.fsi

```diff
@@ -5,7 +5,6 @@ namespace FSharp.Compiler.Diagnostics
 /// Supported kinds of diagnostics by this service.
 [<RequireQualifiedAccess>]
 type FSharpDiagnosticKind =
-    | AddIndexerDot
```

Comment:
@vzarytovskii if by any account this is a breaking change, I will revert it.

Reply:
It is a removal of the public api, so it will be breaking. I would say we should leave it for now. FCS APIs will undergo some cleanups when we'll be doing Analyzers.

Reply:
Alright, reverted